### DISCUSSION TITLE
Place Field

### DISCUSSION THREAD

**Discussion by salierg** [laravel/nova-issues#3761] [Q&A] [answered]
Hi,

Regarding the use of Algolia API for the Nova Place Field, as you know (and as it's mentioned in the nova documentation), Algolia is shutting down is service.
Do you have any plan to fix this issue ? Any alternative with Mapbox or other ?

Thanks

**Comment by crynobone** [comment, marked as answer]
As mentioned in the upcoming 4.0 release upgrade guide: https://nova.laravel.com/docs/4.0/upgrade.html#algolia-place-field 

We have considered the alternative but none offer on par free + request rate available given by Algolia including Mapbox.

**Comment by baszach** [comment]
Hello together,
I've noticed that the `Place` field is still working even though **Algolia Places** should have ended their service three weeks ago. Do you know why that is?
Thanks

**Reply by davidhemphill** [reply]
Companies will often use the sunset date as a "soft" date where you shouldn't rely on the service to work anymore. The actual date when the service stops working will happen randomly when they actually get around to it. 😜
